### PR TITLE
Clarify the role of the system clock in generating the stoken

### PR DESCRIPTION
**Pull request by brendon**
Due to the number of people having problems with expired stokens: https://github.com/ambethia/recaptcha/issues/137 this clarification in the README should help.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,15 @@ Some of the options available:
 You can also override the html attributes for the sizes of the generated `textarea` and `iframe`
 elements, if CSS isn't your thing. Inspect the source of `recaptcha_tags` to see these options.
 
+## v2 API stoken and the System Clock
+
+When this gem interacts with the the reCAPTCHA server using the Version 2 API, we encrypt and provide
+an extra 'secure token'. This enables you to use the Version 2 API on any domain that you like without
+having to register it first with reCAPTCHA. See: https://github.com/ambethia/recaptcha/issues/130
+
+It's **important to note** that a timestamp is encrypted and sent along with a random session_id so your
```
Comment (grosser) [inline README.md]: emphasis on the important part would be nice instead of on the 'important to note' :D

**Comment by grosser** [comment]
simplified it a bit on master :)
thx for the PR!

**Comment by brendon** [comment]
Thanks for that :) Looks good to me. It'll be interesting to see if we still get issues raised for this problem (people not reading the README first) :)

**Comment by grosser** [comment]
The readme is not there to inform people, just to make them feel silly if you tell them it's in the readme ;)

**Comment by brendon** [comment]
Haha, nice!

**Pull request merged**